portal/server/generateApp.ts

```diff
@@ -95,17 +102,21 @@ const generateApp = () => {
   app.use(cookieParser());
 
   app.use(copyCsrfTokenFromQueryToBody());
-  app.use(csrf({
-    cookie: {
-      ...cookie,
-      maxAge: 43200, // 12 hours
-    },
-  }));
+  app.use(
+    csrf({
+      cookie: {
+        ...cookie,
+        maxAge: 43200, // 12 hours
+      },
+    }),
+  );
   app.use(csrfToken());
 
-  app.use(morgan('dev', {
-    skip: (req) => req.url.startsWith('/assets') || req.url.startsWith('/main.js'),
-  }));
+  app.use(
+    morgan('dev', {
```

Comment:
Should `dev` not be picked from `NODE_ENV`?

Reply:
This seems to be an option on the middleware
https://expressjs.com/en/resources/middleware/morgan.html
> Concise output colored by response status for development use. The :status token will be colored green for success codes, red for server error codes, yellow for client error codes, cyan for redirection codes, and uncolored for information codes.